src/base/static/components/app.tsx

```diff
@@ -301,7 +301,18 @@ class App extends Component<Props, State> {
     this.props.loadPagesConfig(resolvedConfig.pages);
     this.props.loadNavBarConfig(resolvedConfig.nav_bar);
     this.props.loadCustomComponentsConfig(resolvedConfig.custom_components);
-    this.props.loadMapStyle(resolvedConfig.map, resolvedConfig.datasets);
+    this.props.loadMapStyle({
+      mapConfig: resolvedConfig.map,
+      datasetsConfig: resolvedConfig.datasets,
+      analysisTargets: resolvedConfig.place.place_detail
+        .filter(detail => detail.geospatialAnalysis)
+        .map(detail => detail.geospatialAnalysis)
+        .reduce((flat, toFlatten) => flat.concat(toFlatten), [])
```

Comment:
The `map` operation above produces nested arrays since the value `geospatialAnalysis` for each category is itself an array to support multiple analyses per category. So the `reduce` flattens everything to a single array.

Reply:
But two `maps`s, a `reduce` and a `filter` just to prep the `loadMapStyle` call seems like a little much-- I'm hoping we can simplify this.

Reply:
Ah, I see :+1: 

I think this could be improved by passing in `resolvedConfig.place.place_detail` to our `loadMapStyle` function directly, and having the duck responsible for performing the ETL of the "place detail config" into an "analysis target" data structure.

We currently have a schema for a "place detail config", but what is an "analysis target"? I don't think "analysis target" should be a concern of App.tsx, nor exposed in the API of `loadMapStyle`. Soon we'll have interfaces for these data structures, and I think "analysis target" should be encapsulated within the `loadMapStyle` function. It's one less data structure that we have to name and keep track of.